ERCS/erc-7548.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 7548
+title: Open IP Protocol
```

Comment:
It would be good to include something about NFTs in the title. Right now it look like a networking protocol ;)